src/sync.ts

```diff
@@ -82,6 +82,27 @@ export class Sync {
         localConfig.customConfig.token,
         localConfig.customConfig.githubEnterpriseUrl
       );
+
+      if (!localConfig.extConfig.forceUpload) {
+        if (
+          await github.CheckIfGistIsNewer(
+            localConfig.extConfig.gist,
+            new Date(localConfig.customConfig.lastUpload)
+          )
+        ) {
+          const message = await vscode.window.showInformationMessage(
+            localize("cmd.updateSettings.warning.gistNewer"),
+            "Yes"
+          );
+          if (message === "Yes") {
+            localConfig.extConfig.forceUpload = true;
+            await state.commons.SaveSettings(localConfig.extConfig);
+          } else {
```

Comment:
We need to show status bar message as
`Sync : Upload Process Cancelled` for sometime (maybe 3-5 seconds)

for the else part.

Reply:
Done